tests/test_quickbooks_all_fields.py

```diff
@@ -246,7 +246,9 @@ class TestQuickbooksAllFields(TestQuickbooksBase):
     }
 
     def name(self):
-        return "tap_tester_quickbooks_combined_test"
+        # Quickbooks uses the token chaining to get the existing token which requires all tests to have same name
+        # So do not overwrite the test name below
```

Comment:
make the same change here as well, use docstring

https://peps.python.org/pep-0257/